src/npm-utils.js

```diff
@@ -131,14 +130,10 @@ module.exports = {
    * @return {Promise} The promise object.
    */
   downloadTarball: function(pkg, version, dir) {
-    var oldCWD = process.cwd();
     process.chdir(dir);
     return execPackCommand([pkg + '@' + version])
       .then(function(filename) {
         return path.resolve(dir || process.cwd(), filename[0]);
-      })
-      .finally(function() {
```

Comment:
Ok, I read your comment in the issue you reported, I understand why you did that.